docs/eth_subscribe.md

```diff
@@ -0,0 +1,110 @@
+## eth_subscribe
+
+Creates a new subscription for desired events. Sends data as soon as it occurs.
+
+### Event Types
+
+- pendingUserOps - user ops validated and put in the mempool
+- submittedUserOps - user ops successfully submitted on chain
+
+### Examples:
+
+### Request
+
+```json
+{
+  "method": "eth_subscribe",
```

Comment:
Should we rename this as `skandha_susbcribe`? 

This is skandha specific endpoints, so let's prefix it with `skandha_`